src/pybkgmodel/processing.py

```diff
@@ -908,3 +909,40 @@ def bkg_map_maker(self, maker):
         if not isinstance(maker, ExclusionMap):
             raise TypeError(f"Maker must be of type {ExclusionMap}")
         BkgMakerBase.bkg_map_maker.fset(maker)
```

Comment:
Here and below: I think this can be simplified to `BkgMakerBase.bkg_map_maker = maker`